config/initializers/csp.rb

```diff
@@ -1,11 +1,11 @@
-SecureHeaders::Configuration.default do |config|  
+SecureHeaders::Configuration.default do |config|
   config.csp = {
     preserve_schemes: true, # default: false.
     default_src: %w('self'),
-    script_src: %w('self' 'unsafe-eval'),
+    script_src: %w('self' 'unsafe-eval' 'unsafe-inline'), #TODO find workaround
```

Comment:
Can you remove the `unsafe-inline` here so we don't leave ourselves vulnerable while we sort out the nonce / csp stuff? After that we're good to merge!